Signal/src/view controllers/CallViewController.swift

```diff
@@ -147,19 +148,36 @@ class CallViewController: UIViewController {
     var thread: TSContactThread!
     var call: SignalCall!
 
-    @IBOutlet weak var contactNameLabel: UILabel!
-    @IBOutlet weak var contactAvatarView: AvatarImageView!
-    @IBOutlet weak var callStatusLabel: UILabel!
+    // MARK: Layout
 
-    // MARK: Outgoing or Accepted Call Controls
+    var hasConstraints = false
 
-    @IBOutlet weak var callControls: UIView!
-    @IBOutlet weak var muteButton: UIButton!
-    @IBOutlet weak var speakerPhoneButton: UIButton!
+    // MARK: Background
+
+    var blurView: UIVisualEffectView!
+
+    // MARK: Contact Views
+
+    var contactNameLabel: UILabel!
+    var contactAvatarView: AvatarImageView!
+    var callStatusLabel: UILabel!
+
+    // MARK: Ongoing Call Controls
+
+    var ongoingCallView: UIView!
+
+    var hangUpButton: UIButton!
+    var muteButton: UIButton!
+    var speakerPhoneButton: UIButton!
+    // TODO: Which call states does this apply to?
+    var textMessageButton: UIButton!
```

Comment:
Okay, for now I've added a "text message" button to the incoming mode. Let's revisit the video button when we get to video.